packages/web/src/pages/settings-page/components/desktop/ManagerMode/AccountsYouManageHomePage.tsx

```diff
@@ -0,0 +1,126 @@
+import { useCallback, useContext, useEffect } from 'react'
+
+import {
+  useApproveManagedAccount,
+  useGetManagedAccounts,
+  useRemoveManager
+} from '@audius/common/api'
+import { Status, UserMetadata } from '@audius/common/models'
+import { accountSelectors } from '@audius/common/store'
+import { Box, Divider, Flex, Text, TextLink } from '@audius/harmony'
+
+import LoadingSpinner from 'components/loading-spinner/LoadingSpinner'
+import { ToastContext } from 'components/toast/ToastContext'
+import { useSelector } from 'utils/reducer'
+
+import { AccountListItem } from './AccountListItem'
+import { sharedMessages } from './sharedMessages'
+import { AccountsYouManagePageProps, AccountsYouManagePages } from './types'
+const { getUserId } = accountSelectors
+
+const messages = {
+  takeControl:
+    'Take control of your managed accounts by making changes to their profiles, preferences, and content.',
+  noAccounts: 'You don’t manage any accounts.',
+  somethingWentWrong: 'Something went wrong. Please try again later.'
+}
+
+export const AccountsYouManageHomePage = ({
+  setPage
+}: AccountsYouManagePageProps) => {
+  const userId = useSelector(getUserId) as number
+  const { data: managedAccounts, status } = useGetManagedAccounts({ userId })
+  const [approveManagedAccount, approveResult] = useApproveManagedAccount()
+  const [rejectManagedAccount, rejectResult] = useRemoveManager()
+  const { toast } = useContext(ToastContext)
+
+  const handleStopManaging = useCallback(
+    ({ userId }: { userId: number; managerUserId: number }) => {
+      setPage(AccountsYouManagePages.STOP_MANAGING, { user_id: userId })
+    },
+    [setPage]
+  )
+
+  const handleApprove = useCallback(
+    ({
+      currentUserId,
+      grantorUser
+    }: {
+      currentUserId: number
+      grantorUser: UserMetadata
+    }) => {
+      approveManagedAccount({ userId: currentUserId, grantorUser })
+    },
+    [approveManagedAccount]
+  )
+
+  const handleReject = useCallback(
```

Comment:
Do we need to create wrapper functions?

Reply:
I figured we might want to add some event tracking here later. Also in this guy we have to do some minor data massaging to shape the request